uwrt_mars_rover_pid_tuning_api/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uwrt_mars_rover_pid_tuning_api)
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+        message_generation
+        roscpp
+        uwrt_mars_rover_hw_bridge
+        uwrt_mars_rover_msgs
+        uwrt_utils
+        )
+
+catkin_package(
+        INCLUDE_DIRS
+        include
+        LIBRARIES
+        ${PROJECT_NAME}
+        CATKIN_DEPENDS
+        roscpp
+        uwrt_mars_rover_hw_bridge
+        uwrt_mars_rover_msgs
+        uwrt_utils
+)
+
+include_directories(
+        include
+        ${catkin_INCLUDE_DIRS}
+)
+
+## Declare a C++ library
+add_library(${PROJECT_NAME}
+        src/${PROJECT_NAME}.cpp
+        )
+
+## Declare a C++ executable
+add_executable(${PROJECT_NAME}_node src/${PROJECT_NAME}_node.cpp)
+
+## Rename C++ executable without prefix
+set_target_properties(${PROJECT_NAME}_node PROPERTIES OUTPUT_NAME node PREFIX "")
+
+## Add cmake target dependencies of the executable
+add_dependencies(${PROJECT_NAME}_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+## Specify libraries to link a library or executable target against
+target_link_libraries(${PROJECT_NAME}_node 
+        ${PROJECT_NAME}
+        ${catkin_LIBRARIES}
+        )
+
+## Mark executables for installation
+install(TARGETS ${PROJECT_NAME}_node
+        RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+        )
+
+## Mark libraries for installation
+install(TARGETS
```

Comment:
is the library going to be used anywhere else?

Reply:
I don't think so. I kind of followed the CMakeLists.txt of NeoPixel as a template.